docs/CMakeLists.txt

```diff
@@ -1,9 +1,7 @@
 # This is the CMake file for building the docs directory of NCEPLIBS-bufr.
-#
-# Ed Hartnett 10/16/20
 
-set(abs_top_srcdir "${CMAKE_SOURCE_DIR}")
-set(config_srcdir "${CMAKE_CURRENT_BINARY_DIR}/../src")
+set(package_base "${CMAKE_SOURCE_DIR}")
```

Comment:
It is safer to use `PROJECT_SOURCE_DIR`

Reply:
OK @aerorahul, I've got everything else ready - just let me know about CMAKE_SOURCE_DIR vs. PROJECT_SOURCE_DIR in src/CMakeLists.txt and docs/CMakeLists.txt, and then I'll push up a new commit.  I assume we should use the same variable name in both places for consistency(?)

@jack-woollen, thanks for the approval, but just FYI that after I push up this next commit for @aerorahul, I'll need you or someone else to also approve those changes as well before I can merge ;-)

Reply:
It is generally safer to use the `PROJECT_` variables when referring to top-level paths when a project is defined.  
Either works in most cases, but the `PROJECT_` works (and `CMAKE_` fails) when a project is included as a submodule or dependency in other projects.

Reply:
Thanks, I was actually just doing a little more reading on that myself.  From what I read, `CMAKE_SOURCE_DIR` is the directory containing the top-level CMakeLists.txt, whereas `PROJECT_SOURCE-DIR` is the directory containing the most-recently executed project() command.  So another way of saying the same thing.

I'll go ahead and push up the new updates